src/Policeman/Core/Package.hs

```diff
@@ -13,17 +21,69 @@ module Policeman.Core.Package
     , ModuleStructure (..)
     ) where
 
+import qualified Data.Text as Text
+
 
 {- | Package name, like @relude@.
 -}
 newtype PackageName = PackageName
     { unPackageName :: Text
     }
 
-newtype Version = Version
-    { unVersion :: Text
+{- | Package version that should satisfy the PVP.
+<https://pvp.haskell.org/>
+-}
+data Version = Version
+    { versionA :: !Int  -- ^ Marketing major.
+    , versionB :: !Int  -- ^ Major.
+    , versionC :: !Int  -- ^ Minor.
+    , versionD :: !Int  -- ^ Tiny minor.
     }
 
+nullVersion :: Version
+nullVersion = Version
+     { versionA = 0
+     , versionB = 0
+     , versionC = 0
+     , versionD = 0
+     }
+
+{- | Shows 'Version' in the standard format:
+
+@
+A.B.C.D
+1.0.2.1
+@
+-}
+versionToText :: Version -> Text
```

Comment:
We will need a function to get `Version` from `[Int]` or from `Text` :thinking: Can you create an issue to write such function and introduce property-based tests for it?

Reply:
Yes, sure :+1:  done here: https://github.com/kowainik/policeman/issues/20